packages/library-base/src/commonMain/kotlin/io/realm/kotlin/Configuration.kt

```diff
@@ -165,6 +165,28 @@ public interface Configuration {
      */
     public val initialDataCallback: InitialDataCallback?
 
+    /**
+     * Enum describing how the Realm file is persisted.
+     */
+    public enum class Durability {
```

Comment:
Why do we have the public concept of `Durability` but set this via `inMemory`  instead of either setting the `durability(IN_MEMORY)` on the configuration or just returning the configuration state as a getter `isInMemory()`?

Reply:
Good points. Just having a `isInMemory()` is a lot smaller change. Exposing the enum would mostly be valuable if we had more than 2 types, which we don't have any plans for adding.